.circleci/config.yml

```diff
@@ -522,6 +522,8 @@ jobs:
             asan.test_wasm_worker_hello
             asan.test_externref_emjs_dynlink
             asan.test_asyncify_longjmp
+            asan.test_emscripten_atomics
+            asan.test_emscripten_futexes
```

Comment:
Is it important to add these to CI here, rather than depend on the test suites bot?

Reply:
I mostly just wanted to prove via CI that these do pass.  I can revert these.